Automattic/jetpack
# Social | Update connections UI initial state for WPCOM

Extracted from #37531 to its own PR.

## Proposed changes:

* Update connections UI initial state for WPCOM

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Ensure that `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );`  is enabled or add the blog sticker to enable the feature.
* Apply this PR to your sandbox
* Goto post editor
* Confirm that connections UI works as expected
